frontend/playwright.config.ts

```diff
@@ -11,7 +11,7 @@ const config: PlaywrightTestConfig = {
 	testDir: './tests',
 	use: {
 		trace: 'retain-on-failure',
-		baseURL: process.env.FRONTEND_API_ENDPOINT,
+		baseURL: process.env.PLAYWRIGHT_TEST_BASE_URL,
```

Comment:
```ts
baseURL: process.env.PLAYWRIGHT_TEST_BASE_URL || "http://localhost:3301",
```

That way we don't have to set the env if we have SigNoz running locally.

Reply:
yeah we can do that let me add this one